routers/tournamentDataRouter.js

```diff
@@ -47,6 +48,7 @@ router.post('/', adminAction, (req, res) => {
       MsLogger.error('Something went wrong while saving data \n' + err)
     })
 
+    mhubConnection.publish(mhubConnection.MHUB_NODES.PUBLIC, 'all:reload')
```

Comment:
What does `all:reload` mean? Does it include the `scores:reload`? Does it include the `teams:reload`? If I write in my code `on('all:reload')`, my code would be harder to read, because no one would know what `all` means. You should publish each of the topics individually